docs/ref/applications.txt

```diff
@@ -239,12 +239,10 @@ Methods
     as registering signals. It is called as soon as the registry is fully
     populated.
 
-    You cannot import models in modules that define application configuration
-    classes, but you can use :meth:`get_model` to access a model class by
-    name, like this::
-
-        def ready(self):
-            MyModel = self.get_model('MyModel')
+    This is the first point in the application loading process where it's safe 
+    to import models (see :ref:`how-applications-are-loaded` for more details).
+    You can do so with a regular ``import`` statement or by using
```

Comment:
Since this is the "first point... where it's safe to import models", you should not import them at an earlier point. This is also emphasized later, in italics, in the section linked to.

But I can keep closer to the original if you like. I found the "in modules" expression to be ambiguous, so let me edit it to specify "module-level"...